CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ For migration information, you can always have a look at https://liip-drifter.re
 
 ## Unreleased
 
+- Update `package.json.webpack.j2` dependencies & config file
```

Comment:
Please add a title above this change: `### Changed`

Also start your line by saying which role this concerns, for example:
`- Webpack role: update dependencies, especially upgrade to Babel 7`